docs/asyncapi.yaml

```
@@ -24,11 +24,29 @@ components:
       summary: |
         Contains the state of an asset as it was stored in a point
         of time in Vulcan.
+      headers: 
+        $ref: "#/components/schemas/assetMetadata"
       contentType: application/json
       payload:
         $ref: "#/components/schemas/assetPayload"
 
   schemas:
+    assetMetadata:
+        type: object
+        additionalProperties: false
+        properties:
+          identifier:
+            type: string
+          type: 
+            type: string
```

I've just realized. Shouldn't this be like follows?

```suggestion
          asset_type:
            $ref: "#/components/schemas/assetType"
```

So it is consistent with `assetPayload`.

---

Rergarding pointing to the assetType schema, yes it makes absolutely sense, regarding renaming it to ``asset_type``, I will leave it as it is because the ``asset_type``  name it's just to avoid errors when generating the go code,  but, as the headers are not used when generating code, we don't need to avoid using reserved keywords in go as names.